### ISSUE TITLE
Invariant Violation: ReactDOMServer does not yet support Suspense.

### ISSUE THREAD

**Issue by tgfischer** [atlassian-labs/storybook-addon-performance#9]
I have a component that uses `Suspense` for code-splitting. I'm not using server-side rendering. When I try to use this add-on, it throws this error

```
Uncaught (in promise) Invariant Violation: ReactDOMServer does not yet support Suspense.
    at http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:241387:38
    at ReactDOMServerRenderer.render (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:241390:17)
    at ReactDOMServerRenderer.read (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:241223:29)
    at Object.renderToString (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:241782:27)
    at Object.run (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:303094:80)
    at http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:302708:98
    at mark (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:302567:26)
    at http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:302708:75
    at withContainer (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:302831:26)
    at runStaticTask (http://127.0.0.1:9009/vendors~main.b2ffbf23a08dd009f236.bundle.js:302707:89)
```

An example component that should reproduce this:
```
// Example/index.js
import React, { Suspense, lazy } from "react";

const LazyExample = lazy(() => import("./Example"));

const Example = props => (
  <Suspense fallback={null}>
    <LazyExample {...props} />
  </Suspense>
);

export default Example;

// Example/Example.js
import React from "react";

const Example = () => <div>Hello, World!</div>

export default Example;
```

Is there a way to disable the server-side rendering?

**Comment by alexreardon** [comment]
Interesting!

**Comment by alexreardon** [comment]
Closed by #12
